mp4parse_capi/src/lib.rs

```diff
@@ -354,6 +365,9 @@ pub unsafe extern fn mp4parse_get_track_info(parser: *mut mp4parse_parser, track
         return MP4PARSE_ERROR_BADARG;
     }
 
+    // Set default value due to it is allocated in gecko.
```

Comment:
Maybe make this "Initialize fields to default values to ensure all fields are always valid."?